modules/serverless-functions-func-yaml-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-functions-func-yaml-environment-variables_{context}"]
+= Referencing environment variables from func.yaml fields
+
+In any of the fields in `func.yaml`, you can put a reference to an environment variable available in the local environment. This can be useful for avoiding storing sensitive information such as an API key in the function configuration.
```

Comment:
The whole sentence is a little bit wrong. The section is about defining environment variables for the deployed function (container) (Kubernetes concept: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)

Referencing a local environment variables (those that are present on developers laptop) and setting these as Kubernetes environment variables is just one type of usecase, ie `{{ env:ENV_VAR }}` described below

Reply:
Okay, it is a little bit confusing. The section should be named `Referencing local environment variables from func.yaml fields`

And this part is sort of duplicate of item #2 in `envs` section of this doc: https://deploy-preview-34442--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-yaml#envs

Reply:
Changed the title, but what about the first sentence?

You are right about the duplication, but IMO it would be misleading to _not_ include the second item inside the `envs` section, we don't want an incomplete list there.